packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -144,8 +144,8 @@ export async function testMemoryLruPersistence(
 }
 
 /** Clears the persistence in tests */
-export async function clearTestPersistence(): Promise<void> {
-  await IndexedDbPersistence.clearPersistence(TEST_PERSISTENCE_PREFIX);
+export function clearTestPersistence(): Promise<void> {
+  return IndexedDbPersistence.clearPersistence(TEST_PERSISTENCE_PREFIX);
```

Comment:
I feel like this is a quiz on how async/await works, but I'm stumped...  Does your change do anything?

Reply:
No... but I find it weird to use async in a method that only uses a single Promise in its return type. I can turn it into a quiz though!